tms/clickup/views.py

```diff
@@ -26,6 +26,7 @@ def post(self, request):
 class ProjectCreateView(ListCreateAPIView):
     queryset = Project.objects.all()
     serializer_class = ProjectSerializer
+    permission_classes = [IsAuthenticated]
 
```

Comment:
in dev.py if you gave default permission is_authenticated then you don't need to override permission_class 
by default it take is_authenticated